src/components/pages/keyGenerator/KeyGeneratorContent.tsx

```diff
@@ -83,6 +86,8 @@ export class KeyGeneratorContent extends React.PureComponent<any, KeyGenerator>
               <div className="chip-container">
                 {words?.map((word, index) => (<Chip key={index} label={`${index + 1}. ${word}`} />))}
               </div>
+              <Label label={en.content.privateKeyWIF} />
+              <Label label={privateKeyWIF} />
```

Comment:
Wouldn't it better to place it in a multiline field? It should be easier for users to copy it then, I believe